bloom-housing/bloom
# feat: show preference option details on listing form

# Pull Request Template

## Issue Overview

This PR addresses #2752

* This PR targets the first preferences PR


## Description

On the listing form in the preferences and programs add section, we now show an expandable box that includes information (title, description, links) for (1) the root preference and (2) each of its options.

## How Can This Be Tested/Reviewed?

Check out the listing form and open the preferences and programs panel & compare to the [Figma](https://www.figma.com/file/Y8bFd0aSz7eYJ2TCNid6s3/bloom-partners?node-id=5800%3A179278).

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
